pyproject.toml

```diff
@@ -46,13 +44,10 @@ pyyaml-include = ">=1.2"
 questionary = ">=1.8.1"
 typing-extensions = { version = ">=3.7.4,<5.0.0", python = "<3.8" }
 
-[tool.poetry.extras]
-docs = [
-    "mkdocs-material",
-    "mkdocstrings",
-]
+[tool.poetry.group.core.dependencies]
+poethepoet = ">=0.12.3"
```

Comment:
I separated `poethepoet` from the other groups because it is a common prerequisite for development and serving the docs. But it might be too confusing and unnecessary. I've moved it to the dev dependencies group.